indico_themes_canonical/static/js/ubuntu_summit.js

```diff
@@ -0,0 +1,112 @@
+/**
+  Toggles the expanded/collapsed classed on side navigation element.
+
+  @param {HTMLElement} sideNavigation The side navigation element.
+  @param {Boolean} show Whether to show or hide the drawer.
+*/
+function toggleDrawer(sideNavigation, show) {
+  const navigationToggle = document.querySelector(".navigationToggleBtn");
+  if (sideNavigation) {
+    if (show) {
+      navigationToggle.setAttribute("aria-expanded", true);
+
+      sideNavigation.classList.remove("is-drawer-hidden");
+      sideNavigation.classList.add("is-drawer-expanded");
+    } else {
+      navigationToggle.setAttribute("aria-expanded", false);
+
+      sideNavigation.classList.remove("is-drawer-expanded");
+      sideNavigation.classList.add("is-drawer-hidden");
+    }
+  }
+}
+
+// throttle util (for window resize event)
+var throttle = function (fn, delay) {
+  var timer = null;
+  return function () {
+    var context = this,
```

Comment:
I don't think this is reassigned, should use `const`
```suggestion
    const context = this,
```